src/redux/migrations.ts

```diff
@@ -1924,4 +1924,5 @@ export const migrations = {
       feedFirstPage: [],
     },
   }),
+  235: (state: any) => state,
```

Comment:
do we need a migration for this change? it feels odd to update the root schema without incrementing the schema number but this migration doesn't remove the `__typename` property on persisted transactions which is the only thing that changes about the state...the persisted transactions are replaced on app start very quickly anyway?

Reply:
Yes I decided to increase the schema number (and hence the no-op migration) to make this change more explicit.
But we could have skipped it.